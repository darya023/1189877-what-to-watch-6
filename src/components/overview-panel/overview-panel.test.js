import React from 'react';
import {render} from '@testing-library/react';
import OverwiewPanel from './overview-panel';

it(`Should OverwiewPanel render correctly`, () => {
  const {container} = render(
      <OverwiewPanel
        description="Test"
        director="Test"
        starring={[`Name Test`, `Name Test2`]}
        rating={4.5}
        reviewsCount={500}
      />
  );
  expect(container).toMatchSnapshot();
});
