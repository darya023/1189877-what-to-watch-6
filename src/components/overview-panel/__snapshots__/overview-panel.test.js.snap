// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should OverviewPanel render correctly 1`] = `
<div>
  <div
    class="movie-rating"
  >
    <div
      class="movie-rating__score"
    >
      4.5
    </div>
    <p
      class="movie-rating__meta"
    >
      <span
        class="movie-rating__level"
      >
        Normal
      </span>
      <span
        class="movie-rating__count"
      >
        500
         ratings
      </span>
    </p>
  </div>
  <div
    class="movie-card__text"
  >
    <p>
      Test
    </p>
    <p
      class="movie-card__director"
    >
      <strong>
        Director: 
        Test
      </strong>
    </p>
    <p
      class="movie-card__starring"
    >
      <strong>
        Starring: 
        Name Test, Name Test2
         and other
      </strong>
    </p>
  </div>
</div>
`;
